### ISSUE TITLE
Latest version JSON

### ISSUE THREAD

**Issue by yofu1234** [javiersantos/AppUpdater#84]
for some reason the library has it that latestVersion integer you put in JSON has to be: latestVersion - 1

"latestVersion": "2.14",

 I.E.: the latest version right now is 2.15. For the pop up to not show up on version 2.15. You have to put latest version as 2.14

Json here: https://github.com/yofu1234/androidappupdate/blob/master/update-changelog.json

**Comment by javiersantos** [comment]
I don't see the problem here.

When I use 2.15 as the `versionName` and upload the next JSON to my server, I get "No updates available".

```json
{
  "latestVersion": "2.15",
  "url": "https://play.google.com/store/apps/details?id=com.stackkedteam.feedshare",
  "releaseNotes": [
    "1. Added image compression so uploading images are now way faster.",
    "2. Moar Bug fixes"
  ]
}
```

**Comment by yofu1234** [comment]
What does your code look like?

I'm not using 	
`.setContentOnUpdateNotAvailable("No update available. Check for updates again later!")
`

because I don't want a pop up when no code is available.


Here is my code

`        AppUpdater appUpdater = new AppUpdater(this)
                .setTitleOnUpdateAvailable("Important: Must Update")
                .setButtonUpdate("Update now!")
                .setButtonDismiss("Later")
                .setButtonDoNotShowAgain(null)
                .setIcon(R.mipmap.ic_launcher) // Notification icon
                .setUpdateFrom(UpdateFrom.JSON)
            .setUpdateJSON("https://raw.githubusercontent.com/yofu1234/androidappupdate/master/update-changelog.json");
        appUpdater.start();`

**Comment by javiersantos** [comment]
Is this issue fixed with the latest update and #85?

**Comment by yofu1234** [comment]
Yup, the new update fixed the issue! Thanks!
